### ISSUE TITLE
Setup fails if Adonis/Src/View is not present (--api-only)

### ISSUE THREAD

**Issue by borie88** [dysfunctio-nl/adonis-watchtower#3]
I am trying to install watchtower on an Adonis installation using the --api-only flag. When I perform adonis watchtower:setup, it fails with the error, Error: Cannot find module 'Adonis/Src/View'. Is there a workaround for this when edge is not installed?

**Comment by eNzyOfficial** [comment]
I am out of the country at the moment, I should be back next week and I can take a look at fixing it.

The problem will be this function call:

https://github.com/dysfunctio-nl/adonis-watchtower/blob/f0c0a96e158c15e457b53c394a9df84c975a3b06/providers/WatchTowerProvider.js#L42

Temporarily, you can make your own provider, replicating the watchtower provider and removing the view related function and use that version instead of the default watchtower provider and it should work.

**Comment by borie88** [comment]
Yes, thanks. I commented out anything needing the package and it works. Thank you
